glm/README.xoreos

```diff
@@ -5,3 +5,4 @@ it:
 
 - Modified include files to silence compiler warnings about variable
   shadowing
+- Commented out GLM_STATIC_ASSERT lines to silence Travis CI warnings
```

Comment:
Hmm, a better, less intrusive way would be to undefine and redefine GLM_HAS_STATIC_ASSERT to 0 in setup.hpp.

However, it'd really like to keep these asserts. I'd rather disable `-Wunused-local-typedef`. See how we disable `-Wundefined-var-template` in CMakeLists.txt:104 and configure.ac:135 for an example on how that's done.